waspc/data/Generator/templates/server/src/_types/index.ts

```diff
@@ -49,6 +67,20 @@ type PrismaDelegate = {
   {=/ entities =}
 }
 
+type Context<Entities extends _Entity[]> = Expand<{
+  entities: Expand<EntityMap<Entities>>
+}>
+
+type ContextWithUser<Entities extends _Entity[]> = Expand<Context<Entities> & UserInContext>
+
+export type UserInContext = {
+  // TODO: This type must match the logic in core/auth.js (if we remove the
```

Comment:
Just to check, do we also have corresponding comment there?

Reply:
This was Filip's original comment I just moved. But yes, just checked that file and there is one there too 👍🏻